hardhat.config.ts

```diff
@@ -1,6 +1,7 @@
 
 import { HardhatUserConfig } from "hardhat/config";
 
+import "hardhat-storage-layout";
```

Comment:
What is this for?

Reply:
The command it enables is `npx hardhat storage-layout`